x-pack/platform/plugins/shared/onechat/public/application/components/tools/esql/esql_tool.tsx

```diff
@@ -107,6 +111,28 @@ export const EsqlTool: React.FC<EsqlToolProps> = ({
     </EuiButton>
   );
 
+  const saveAndTestButton = (
```

Comment:
do we always need to save and test? e.g. if no changes to input form is made (form `formState.isDirty` property is false) we can just display `Test` rather than save and test?